ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Node.hs

```diff
@@ -468,7 +522,8 @@ instance CardanoHardForkConstraints c
       [ (NodeToNodeV_7, CardanoNodeToNodeVersion5)
       , (NodeToNodeV_8, CardanoNodeToNodeVersion5)
       , (NodeToNodeV_9, CardanoNodeToNodeVersion6)
-      , (NodeToNodeV_10, CardanoNodeToNodeVersion6)
+      , (NodeToNodeV_10, CardanoNodeToNodeVersion7)
+      , (NodeToNodeV_11, CardanoNodeToNodeVersion7) -- TODO: check this!
```

Comment:
I think the `-- TODO: check this!` can be removed -- this mapping looks correct to me.

... But the tuple above should stay as `(NodeToNodeV_10, CardanoNodeToNodeVersion6)`, right?

Reply:
I think so, but I'll confirm with @coot

Reply:
Yes, let's not modify what `NodeToNodeV_10` (it will be released in `1.35.5`).

We also introduce `NodeToNodeV_11` in #4019 - just to note that there will be a conflict on one side or the other.

Reply:
I just Approved this PR; this comment is the only one that wasn't Resolved. I'll let @dnadales resolve it, just to make sure he has seen Marcin's warning.